repos/system_upgrade/el7toel8/models/fstabsignal.py

```diff
@@ -0,0 +1,6 @@
+from leapp.models import Model
+from leapp.topics import BootPrepTopic
+
+
+class FstabSignal(Model):
```

Comment:
this is nothing telling model. whoever will see it, it will miss the purpose of the model. Missing docstring, model name is not intuitive.